catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java

```diff
@@ -161,7 +170,7 @@ private InputStream getInputStreamFromResource(
       } else if (resource.matches("((?i)cities[0-9]+)")) {
         resource = resource.toLowerCase();
         // Support case insensitive country codes
-      } else if (!resource.equalsIgnoreCase("allCountries")) {
+      } else if (!resource.equalsIgnoreCase("allCountries") && !urlValidator.isValid(resource)) {
```

Comment:
❓ Do you need to check if it is a valid URL here if you already know that the string is "allCountries"?